appvm-scripts/usrbin/qubes-run-xephyr

```diff
@@ -2,4 +2,10 @@
 DISPLAY_XEPHYR="$1"
 OPTIONS_XEPHYR="-no-host-grab -fullscreen"
 
-LD_PRELOAD=/usr/lib64/qubes-gui-daemon/shmoverride.so /usr/bin/Xephyr $OPTIONS_XEPHYR "$DISPLAY_XEPHYR" > ~/.xephyr-errors 2>&1
+if [ -e /etc/debian_version ]; then
```

Comment:
Better check shmoverride.so existence directly.

Reply:
I adjusted with existence in both cases. In any other case, that will pass empty/env value.